package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.2.25",
```

Comment:
The tag 1.2.26 could not be reused, and I saw that the repo in prod had the tag `1.2.26` (link: [EDS Release Tags history](https://github.com/getethos/ethos-design-system/releases)). It seemed to me the version should be bumped to match the tag?

Reply:
I confirmed from the diff of the last release that the tag had been updated to 1.2.26 but not the version. This seems like a mistake.
Link: https://github.com/getethos/ethos-design-system/pull/445/files